pkg/resource/interfaces.go

```diff
@@ -216,18 +233,12 @@ type Workload interface {
 type Composite interface {
 	Object
 
-	Conditioned
-	Bindable
```

Comment:
Isn't `Bindable` necessary for the relation between namespaced publication and composite resource?

Reply:
Alright, just checked the design and remembered it was under `spec`. I feel like we should change the implementation instead of the interface since the function signatures will be the same but not so sure. Anyway, you'd add it when you implement the publication.

Reply:
@muvaf I thought we agreed to remove `bindingPhase` altogether?